frontend/app/dashboard/src/views/dashboard/settings/modules/members/ImportMembersQuestionsView.vue

```diff
@@ -172,467 +214,412 @@
 
                 <LoadingButton :loading="saving">
                     <button type="button" class="button primary" @click="goNext">
-                        Importeer {{ members.length }} leden
+                        Importeer {{ $members.length }} leden
                     </button>
                 </LoadingButton>
             </template>
         </STToolbar>
     </div>
 </template>
 
-<script lang="ts">
+<script lang="ts" setup>
 import { AutoEncoder, AutoEncoderPatchType, PatchableArray, PatchableArrayAutoEncoder } from '@simonbackx/simple-encoding';
-import { ComponentWithProperties, NavigationMixin } from "@simonbackx/vue-app-navigation";
-import { Component, Mixins, Prop } from "@simonbackx/vue-app-navigation/classes";
-import { BackButton, Checkbox, Dropdown, ErrorBox, LoadingButton, Radio, RadioGroup, STErrorsDefault, STInputBox, STList, STListItem, STNavigationBar, STToolbar, Toast, Validator } from "@stamhoofd/components";
+import { ComponentWithProperties, useCanDismiss, useDismiss, usePresent } from "@simonbackx/vue-app-navigation";
+import { Dropdown, ErrorBox, LoadingButton, Radio, RadioGroup, STErrorsDefault, STInputBox, STList, STListItem, STNavigationBar, STToolbar, Toast, useOrganization } from "@stamhoofd/components";
 import { Gender, Group, Organization, OrganizationPatch, Parent, ParentTypeHelper, Registration } from "@stamhoofd/structures";
 import { Formatter, Sorter } from '@stamhoofd/utility';
-
 import { FamilyManager } from '../../../../../classes/FamilyManager';
 import { ImportingMember } from '../../../../../classes/import/ImportingMember';
 import ImportAutoAssignedView from './ImportAutoAssignedView.vue';
+import { Ref, ref, computed, onMounted } from 'vue';
+import { useLegacyMemberManager } from '@stamhoofd/registration';
+
+const props = defineProps<{ members: ImportingMember[] }>();
+const $members = ref(props.members) as Ref<ImportingMember[]>;
```

Comment:
```suggestion
const $members = computed(() => props.members) as Ref<ImportingMember[]>;
```
Denk dat we ofwel computed moeten gebruiken, of overal props.members, want anders vang je geen veranderingen in properties